src/modules/custodian/custodian.service.test.ts

```diff
@@ -82,6 +135,31 @@ describe('CustodianService', () => {
 
       expect(httpServicePost).toHaveBeenCalledTimes(1);
       expect(httpServicePost).toHaveBeenCalledWith(...expectedHttpServicePostArgs);
+
+      expect(cacheManagerSet).toHaveBeenCalledTimes(2);
```

Comment:
I would argue that this `it` case perhaps tests too much and we should maybe split this off into a different test (e.g., `it calls the set method on the cache manager correctly`

Reply:
Made test more readable, but left same number of assessments. They test caching of one user journey, for me looks like they all are required.